Assets/Scripts/Utilities/Logger.cs

```diff
@@ -30,7 +30,7 @@ public static void LogException(Exception exception)
         }
     }
 
-    public static void LogAssertion(string message)
+    public static void LogAssertion(object message)
```

Comment:
It has both actualiy, so you can use whatever you prefer. Again, the point here is not to create the best logger, that we can do later, this is supposed be a carbon copy of Unity's logging. It will be difficult to get this merged as it is. If, and only if, this is merged should we look into going above and beyond.